src/dlsproto/client/request/internal/GetRange.d

```diff
@@ -633,7 +633,7 @@ private scope class GetRangeHandler
         private bool waitForRecords ()
         {
             // Wait for the next batch, unless we already have one.
-            if ((*this.buffers.input).length == 0)
+            if (this.buffers.empty())
```

Comment:
Yes. Oops.

Reply:
I guess this fixes the bug revealed in testing?

Reply:
No, it haven't tested it yet, but I'm not so confident this will fix that bug.